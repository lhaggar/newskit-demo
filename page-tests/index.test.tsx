import IndexPage from '../pages';
import { render } from '@testing-library/react';

describe('Index Page', () => {
  it('should render as expected', () => {
    const r = render(<IndexPage />).asFragment();
    expect(r).toMatchSnapshot();
  });
});
