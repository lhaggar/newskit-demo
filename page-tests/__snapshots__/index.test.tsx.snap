// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page should render as expected 1`] = `
<DocumentFragment>
  <div
    class="css-1lecspc"
  >
    <div
      class="css-1r3hoet"
    >
      <h1
        class="css-96okl9"
      >
        NewsKit Demo
      </h1>
    </div>
    <div
      class="css-q8gelu"
    >
      <h2
        class="css-6phmxu"
      >
        v5.1.1
      </h2>
    </div>
    <div
      class="css-1gojmfd"
    >
      <hr
        aria-hidden="true"
        class="css-1aqwza0"
        data-testid="divider"
      />
    </div>
    <div
      class="css-oqz0e1"
    >
      <div
        class="css-q8gelu"
      >
        <div
          class="css-1n7m8q9"
        >
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page1"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Step 1: Figma Design
                </span>
              </span>
            </a>
          </div>
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page2"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Step 2: Basic Components
                </span>
              </span>
            </a>
          </div>
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page3"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Step 3: Setting Type Presets
                </span>
              </span>
            </a>
          </div>
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page4"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Step 4: Layout
                </span>
              </span>
            </a>
          </div>
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page5"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Step 5: Setting Style Presets
                </span>
              </span>
            </a>
          </div>
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page6"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Step 6: Custom Component
                </span>
              </span>
            </a>
          </div>
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page7"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Step 7: Theme
                </span>
              </span>
            </a>
          </div>
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page8"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Step 8: Full Page Example
                </span>
              </span>
            </a>
          </div>
          <div
            class="css-1a047cs"
          >
            <a
              class="css-te0m4d"
              href="/page9"
            >
              <span
                class="css-7eo7xt"
              >
                <span
                  class="css-1l5ehtz"
                >
                  Theme Checker
                </span>
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="css-q8gelu"
    >
      <hr
        aria-hidden="true"
        class="css-1aqwza0"
        data-testid="divider"
      />
    </div>
    <footer>
      <p
        class="css-10zdq1v"
      >
        <a
          class="css-te0m4d"
          href="https://newskit.co.uk/"
        >
          <span
            class="css-lmlygy"
          >
            <span
              class="css-14dqn90"
            >
              <span
                class="css-1l5ehtz"
              >
                newskit.co.uk
              </span>
            </span>
            <span
              class="css-14dqn90"
            >
              <svg
                class="css-1wvnngw-EmotionIconBase ex0cdmw0"
                fill="currentColor"
                focusable="false"
                role="img"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  External link
                </title>
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
                <path
                  d="M19 19H5V5h7V3H5a2 2 0 00-2 2v14a2 2 0 002 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
                />
              </svg>
            </span>
          </span>
        </a>
         | 
        <a
          class="css-te0m4d"
          href="https://github.com/newscorp-ghfb/newskit"
        >
          <span
            class="css-lmlygy"
          >
            <span
              class="css-14dqn90"
            >
              <span
                class="css-1l5ehtz"
              >
                NewsKit on GitHub
              </span>
            </span>
            <span
              class="css-14dqn90"
            >
              <svg
                class="css-1wvnngw-EmotionIconBase ex0cdmw0"
                fill="currentColor"
                focusable="false"
                role="img"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  External link
                </title>
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
                <path
                  d="M19 19H5V5h7V3H5a2 2 0 00-2 2v14a2 2 0 002 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
                />
              </svg>
            </span>
          </span>
        </a>
         | 
        <a
          class="css-te0m4d"
          href="https://github.com/lhaggar/newskit-demo"
        >
          <span
            class="css-lmlygy"
          >
            <span
              class="css-14dqn90"
            >
              <span
                class="css-1l5ehtz"
              >
                This Demo on GitHub
              </span>
            </span>
            <span
              class="css-14dqn90"
            >
              <svg
                class="css-1wvnngw-EmotionIconBase ex0cdmw0"
                fill="currentColor"
                focusable="false"
                role="img"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  External link
                </title>
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
                <path
                  d="M19 19H5V5h7V3H5a2 2 0 00-2 2v14a2 2 0 002 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
                />
              </svg>
            </span>
          </span>
        </a>
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
